backend/canisters/community/impl/src/lifecycle/inspect_message.rs

```diff
@@ -11,9 +11,7 @@ fn accept_if_valid(state: &RuntimeState) {
 
     // 'inspect_message' only applies to ingress messages so calls to c2c methods should be rejected
     let is_c2c_method = method_name.starts_with("c2c") || method_name == "wallet_receive";
-    let is_frozen = state.data.frozen.value.is_some();
```

Comment:
I removed the handling of Frozen so that anyone calling gets back the correct response rather than the call getting rejected.